src/main/resources/plugin.yml

```
@@ -5,3 +5,12 @@ api-version: 1.15
 prefix: ChatDictionary
 authors: [ zauuuw ]
 description: チャットに辞書機能を追加します
+commands:
+  dictionary:
+    dictionary.command.setting
+  usage:
+    /dictionary <add|remove|options|config reload> ...
```

プラグインのロード時にエラー吐いてますので、修正お願いします。

```suggestion
commands:
  dictionary:
    description: 辞書のワードを追加、削除、設定するためのコマンドです。
    permission: dictionary.command.setting
    usage: /dictionary <add|remove|options|config reload> ...
```